src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
 /*
-* This file is part of the symfony project.
-*
-* (c) Vincent Touzet <vincent.touzet@dotsafe.fr>
```

Comment:
Spotted :rofl:

Reply:
https://github.com/symfony/symfony/pull/30805 for reference

Reply:
hmm.. this file were renamed and git history trapped me. It exists in 4.4 too :(